### ISSUE TITLE
Does Shikhondo really support FlipGrip / how do you enable it?

### ISSUE THREAD

**Issue by silverhook** [Alshie/does-it-flip#3]
I just got Shikhondo: Soul Eater and can’t figure out how to put it into TATE/vertical mode.

If you know how to change the mode, please let me know – or better yet, include that as a text field on the page itself (I’ll open a separate issue for that).

If you can’t, let’s remove it from the list then.

**Comment by silverhook** [comment]
As promised, I created #4, which is how this issue could be solved.

**Comment by silverhook** [comment]
According to [this article](https://retronauts.com/article/946/the-comprehensive-list-of-flip-grip-friendly-vertical-games-for-switch), it does *not* support it. The same goes for a few other games that you list. Did that get fixed in between, or what?

**Comment by Alshie** [comment]
I had added this to the initial list on the basis of this article https://switcher.co/features/vertical-flip-grip-switch-games/

I have removed for now on the back of your feedback and I'm trying to establish the status of tate mode in Shikhondo (although I'm fairly certain it must not be there if you can't find it).

Some of the other titles form the original Retronauts list have had support added in - however I am trying to verify these also. Unfortunately I don't actually own all of the games to verify myself.

I wonder if adding a "source" for the information would be useful - to see where it came from e.g.link to article/tweet etc.

I'm sorry if you wasted money on a title on the back of this. It's exactly what I hoped to avoid, when I was looking for titles I found lots of lists that were not up to date with new releases etc.

**Comment by silverhook** [comment]
I wrote to Digerati (Shikhondo’s publisher) support and they said that TATE/vertical mode is currently not planned, but they’ll let me know if this changes in the future.

Regarding the source of information, I’m split. As we just proved, unless it’s the dev or publisher, the source could be false as well. And in that case I don’t see any better reason on why a visitor to the page would care about the source of the info, other than for shifting the blame to someone else for false information. Perhaps a checkbox for `tested` would be better and then just a small symbol on the page that someone verified it.

Eh, don’t worry, I would have bought Shikhondo anyways, it’s a good game and I got it for a very good discount.

Yeah, buying all FlipGrip-supported games just to list them would be waaaay too much. Let’s crowdsource this. I’m happy to add what I’ve got.
